src/main/resources/default/templates/biz/protocol/errors.html.pasta

```diff
@@ -35,9 +35,9 @@
                                     <div class="col-md-4">@i18n("StoredIncident.user")</div>
                                 </div>
                                 <div class="row">
-                                    <div class="muted col-md-4">@i18n("StoredIncident.firstOccurrence")</div>
-                                    <div class="muted col-md-4">@i18n("StoredIncident.category")</div>
-                                    <div class="muted col-md-4">@i18n("StoredIncident.node")</div>
+                                    <div class="text-muted col-md-4">@i18n("StoredIncident.firstOccurrence")</div>
+                                    <div class="text-muted col-md-4">@i18n("StoredIncident.category")</div>
+                                    <div class="text-muted col-md-4">@i18n("StoredIncident.node")</div>
```

Comment:
**Question:** Nicht auch <small> statt <div>?

Reply:
small in the table-content is too small, text-small will be correct (see last commit)